gp2gp-translator/src/integrationTest/resources/json/LargeMessage/expectedBundleScenario1.json

```diff
@@ -61,11 +61,13 @@
             "value": "B83002"
           }
         ],
-        "type": [
-          {
-            "text": "General practice (organisation)"
-          }
-        ],
+        "type": [ {
+          "coding": [ {
+            "system": "394745000",
+            "code": "2.16.840.1.113883.2.1.3.2.4.15",
```

Comment:
This looks very suspicious.

The two fields look like they're the wrong way around. `394745000` would be the code in this example, and `2.16...` is the system.